### ISSUE TITLE
Deprecate the npm package?

### ISSUE THREAD

**Issue by jimmywarting** [webpack/node-libs-browser#109]
If you are not accepting more feature PR, then maybe it's time to deprecate the npm package? 

> NOTE: This library is deprecated and won't accept Pull Requests that include Breaking Changes or new Features. Only bugfixes are accepted.

**Comment by niksy** [comment]
I agree. `node-libs-browser` is still very useful, but should be deprecated and that should be communicated when installing it.

As alternative, I’ve made [package](https://github.com/niksy/node-stdlib-browser) which forks this one and adds additional and latest Node features, and should also work with Rollup and other bundlers.
